public/javascript/main.js

```diff
@@ -376,5 +376,62 @@ $(function(){
 
           $(this).data('chart', chart);
       }
-  })
+  });
+
+  $(".js-download-with-term").on("click", function(e){
+    var term = $(e.target).data("term");
+    var house = $(e.target).data("house");
+    if (term && house) {
+        // inject query vars into URL before the target anchor
+        e.preventDefault();
+        location = $(e.target).attr('href')
+          .replace('#', "?" + $.param({term: term, house: house}) + "#");
+    }
+  });
+
+  if (typeof pageInitFunction === 'function') {
+      pageInitFunction();
+  }
+
 });
+
+// collapse_displayed_items:
+// If there are too many items being displayed in the list, hide
+// all but a few, making sure the target element (if there is one)
+// is included in those that are not hidden. Add a button for
+// revealing the hidden items.
+// Note this is used on the download page, where (currently) there
+// may be more than one such list (e.g., one for each legislature).
+
+function collapse_displayed_items(
```

Comment:
@davewhiteland Sorry to be *that guy*, but all of the other variables and function names in viewer-sinatra use camelCase. Why doesn't this function (and the variables inside it)?